libraries/Microsoft.Bot.Builder.LanguageGeneration/TemplateEngine.cs

```diff
@@ -86,7 +86,7 @@ public TemplateEngine AddFiles(IEnumerable<string> filePaths, ImportResolverDele
         /// <returns>Template engine with the parsed content.</returns>
         public TemplateEngine AddText(string content, string name, ImportResolverDelegate importResolver)
         {
-            var sources = new Dictionary<string, LGSource>();
+            var sources = new Dictionary<string, LGResource>();
             LoopLGText(content, name, sources, importResolver);
```

Comment:
Can be refined as `Templates = Templates.Concat(sources.SelectMany(u => u.Value.Templates)).ToList();`

Reply:
resolved